libdecor/libdecor-0.1.1.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "libdecor",
+  "buildsystem": "meson",
+  "config-opts": [
+    "-Ddemo=false"
+  ],
+  "sources": [
+    {
+      "type": "git",
+      "url": "https://gitlab.freedesktop.org/libdecor/libdecor.git",
```

Comment:
Could you use release tarballs, they will speed up build times for every project using this.

https://gitlab.freedesktop.org/libdecor/libdecor/uploads/ee5ef0f2c3a4743e8501a855d61cb397/libdecor-0.1.1.tar.xz

Reply:
I believe I've added the source properly now